wtp/apiCaller.js

```diff
@@ -17,7 +17,7 @@ const RUN_TEST_URL = 'http://www.webpagetest.org/runtest.php';
 const GET_TEST_STATUS = 'http://www.webpagetest.org/testStatus.php';
 
 
-const getTestResults = (testId, cb) => {
+const getTestResults = (testId, quality, cb) => {
```

Comment:
Seems like we're passing more and more params to the methods. Perhaps worth considering moving to named parameters using [parameters destructuring](https://simonsmith.io/destructuring-objects-as-function-parameters-in-es6)? Will be more readable.

Reply:
It would be a big overhead to refactor the all code for this. I think if we see that we keep adding parameters it would be worth doing it.

Reply:
What's our magic number? `(imagesArray, batchSize, dpr, metaData, quality, cb, rollBarMsg)` seems long enough already...